main.go

```diff
@@ -395,23 +413,42 @@ func main() {
 
 		log.Infof("Zipalign Build Artifact")
 		signedArtifactName := fmt.Sprintf("%s-bitrise-signed%s", buildArtifactBasename, artifactExt)
-		signedBuildArtifactPaths[i] = filepath.Join(buildArtifactDir, signedArtifactName)
-		if err := zipalignBuildArtifact(zipalign, unalignedBuildArtifactPth, signedBuildArtifactPaths[i]); err != nil {
+		fullPath := filepath.Join(buildArtifactDir, signedArtifactName)
+		switch artifactExt {
+		case "apk":
```

Comment:
if apk is the default, no need for this case, which leads a 2 element switch-case, which is an if-else statement